tests/media/test_pending_media_deletion.py

```diff
@@ -0,0 +1,241 @@
+import os
+from http import HTTPStatus
+
+from twisted.test.proto_helpers import MemoryReactor
+from twisted.web.resource import Resource
+
+from synapse.media.filepath import MediaFilePaths
+from synapse.media.media_repository import MediaRepository
+from synapse.rest import admin
+from synapse.rest.client import login, media, profile, room
+from synapse.server import HomeServer
+from synapse.types import UserID
+from synapse.util import Clock
+from synapse.util.stringutils import (
+    random_string,
+)
+
+from tests import unittest
+
+
+class PendingMediaDeletionTestCase(unittest.HomeserverTestCase):
+    servlets = [
+        media.register_servlets,
+        login.register_servlets,
+        admin.register_servlets,
+        room.register_servlets,
+        profile.register_servlets,
+    ]
+
+    def make_homeserver(self, reactor: MemoryReactor, clock: Clock) -> HomeServer:
+        config = self.default_config()
+        config.update(
+            {
+                "experimental_features": {
+                    "msc3911": {
+                        "enabled": True,
+                        "enable_media_retention": True,
+                    },
+                },
+            }
+        )
+        return self.setup_test_homeserver(config=config)
```

Comment:
Let's try and follow the pattern for `default_config()`. Where we overload the `default_config()` function and call it's `super()`
```suggestion
    def default_config(self) -> JsonDict:
        config = super().default_config()
        config.setdefault("experimental_features", {}).update(
            {
                "msc3911": {
                    "enabled": True,
                    "enable_media_retention": True,
                },
            },
        )
        return config
```

Something like this, but it is only something I look for when there is no other actual setup work being done